src/javascript/ContentManager/ContentRoute/ContentLayout/ContentListTable/ContentListTable.jsx

```diff
@@ -286,6 +291,30 @@ export class ContentListTable extends React.Component {
         return (icon.includes('.png') ? icon : icon + '.png');
     }
 
+    getMediaIcon(node) {
+        let {classes} = this.props;
+        switch (node.primaryNodeType.displayName) {
+            case 'Folder':
+                return <Folder className={classes.icon}/>;
+            case 'File':
+                if (node.mixinTypes.length !== 0 && !_.isEmpty(node.mixinTypes.filter(mixin => mixin.name === 'jmix:image'))) {
+                    return <ImageIcon className={classes.icon}/>;
+                }
+
+                if (node.name.includes('zip') || node.name.includes('tar') || node.name.includes('rar')) {
```

Comment:
You should check the file "extension" only here instead of using "includes" which will match any part of the filename.

Reply:
done with regex